clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags

```diff
@@ -14,5 +14,8 @@ This script triggers diagnostics collection.
 
 The packaged diagnostics are created in /var/clearwater-diags-monitor/dumps/*.tar.gz'
 
-# Write a file to trigger a diagnostic dump.
-echo "Manually triggered by /usr/share/clearwater/bin/gather_diags" > /var/clearwater-diags-monitor/tmp/core.gather_diags.$(date +%s)
+# We call through to the gather_diags_and_report_location script, but with a
+# flag that skips the wait for completion and location reporting, making sure
+# to pass through the --force or --force-confirmed flags as the first arguments
+
+/usr/share/clearwater/bin/gather_diags_and_report_location $@ --no-report
```

Comment:
How are you expecting the usage options to work here?

Reply:
Wouldn't it be simpler for these scripts to work the other way round? I.e. `gather_diags_and_report_location` calls `gather_diags` and then waits for compeltion?

Reply:
Usage options are all passed through, with --no-report added on the end. not sure i follow what you mean in the first comment.
On the second, yes, it probably would be, except this seemed simpler at the time. I'll take a look and see how much i care.